### PR TITLE
EVM-C: Renames

### PR DESCRIPTION
**Pull request by chfast**

### CODE REVIEW

File: libevm/JitVM.cpp [hunk 1]
```diff
@@ -398,25 +397,25 @@ owning_bytes_ref JitVM::exec(u256& io_gas, ExtVMFace& _ext, OnOpFunc const& _onO
...
 		return EVM_CONSTANTINOPLE;
 	if (_schedule.haveRevert)
 		return EVM_BYZANTIUM;
 	if (_schedule.eip158Mode)
-		return EVM_CLEARING;
+		return EVM_SPURIOUS_DRAGON;
 	if (_schedule.eip150Mode)
-		return EVM_ANTI_DOS;
+		return EVM_TANGERINE_WHISTLE;
 	return _schedule.haveDelegateCall ? EVM_HOMESTEAD : EVM_FRONTIER;
```
Comment (axic) [inline libevm/JitVM.cpp]: Would it be easier to read making this thing an if-elseif-else waterfall?

File: libevm/JitVM.cpp [hunk 2]
```diff
@@ -398,25 +397,25 @@ owning_bytes_ref JitVM::exec(u256& io_gas, ExtVMFace& _ext, OnOpFunc const& _onO
 	return output;
 }
 
-evm_mode JitVM::scheduleToMode(EVMSchedule const& _schedule)
+evm_revision JitVM::scheduleToMode(EVMSchedule const& _schedule)
```
Comment (axic) [inline libevm/JitVM.cpp]: Also renaming to `scheduleToRevision`?

File: libevm/JitVM.cpp (hunk 1 above)
Comment (chfast) [inline libevm/JitVM.cpp]: "No else after return". But I can replace the last `? :`.

File: libevm/JitVM.cpp (hunk 2 above)
Comment (chfast) [inline libevm/JitVM.cpp]: Yep.

File: libevm/JitVM.cpp (hunk 1 above)
Comment (axic) [inline libevm/JitVM.cpp]: I do not necessarily agree with the "no else after return" rule, in many cases it makes it more readable.

At least replace the ternary, should help readability.

**Comment by chfast** [comment]
Now it compiles. What do you think @axic?

**Pull request merged**